core/tests/protractor/collections.js

```diff
@@ -82,7 +110,93 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('checks for console errors in a collection with one node', function() {
+    users.login('player@collections.com');
+    browser.get('/search/find?q=');
+    browser.waitForAngular();
+    element.all(by.css(
+      '.protractor-test-collection-summary-tile-title')
+    ).filter(function (elem) {
+      return elem.getText().then(function (text) {
+        return text === 'Test Collection 2';
+      });
+    }).first().click();
+    browser.waitForAngular();
+    general.waitForSystem();
+    browser.getCurrentUrl().then(function(url) {
+      var pathname = url.split('/');
+      COLLECTION_ID = pathname[4];
```

Comment:
Actually, this isn't what I meant. I meant storing collectionId (or even the collection URL) at the end of the beforeEach() phase and then using that to visit the collection URL directly in each of the individual tests, so that you don't need to go through the library or creator dashboard. You can do URLs for both the collection player and collection editor pages.

Reply:
Oh, ok, will try that then.

Reply:
Done.